includes/emails/email-summary/class-edd-email-summary.php

```diff
@@ -69,7 +79,11 @@ public function get_email_subject() {
 		/* Translators: Site domain name */
 		$email_subject = sprintf( __( 'Easy Digital Downloads Summary - %s', 'easy-digital-downloads' ), $this->get_site_url() );
 
-		return apply_filters( 'edd_email_summary_subject', $email_subject );
+		if ( $this->test_mode ) {
```

Comment:
I'm not sure what happened but my test email came through with no subject at all.

Reply:
Interesting I haven't seen that yet.

Reply:
It could be just something weird with how Local/Mailhog work together. If it's only happening to me then we can ignore it. Tinkering with various modifications of the subject line got some options to work and some not.